src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -1301,6 +1305,43 @@ def set_up_network_profile(self, mc: ManagedCluster) -> ManagedCluster:
         mc = super().set_up_network_profile(mc)
         network_profile = mc.network_profile
 
+        ip_families = self.context.raw_param.get('ip_families')
+
+        if ip_families:
+            network_profile.ip_families = ip_families.split(',')
+
+        pod_cidrs = self.context.raw_param.get('pod_cidrs')
+        if pod_cidrs:
+            network_profile.pod_cidrs = pod_cidrs.split(',')
+
+        service_cidrs = self.context.raw_param.get('service_cidrs')
+        if service_cidrs:
+            network_profile.service_cidrs = service_cidrs.split(',')
+
+        ipv6_count = self.context.raw_param.get('load_balancer_managed_outbound_ipv6_count')
+        if ipv6_count:
+            print('pre + ' + str(network_profile))
+            if network_profile.load_balancer_profile:
+                network_profile.load_balancer_profile = update_load_balancer_profile(
+                    self.context.get_load_balancer_managed_outbound_ip_count(),
+                    ipv6_count,
+                    self.context.get_load_balancer_outbound_ips(),
+                    self.context.get_load_balancer_outbound_ip_prefixes(),
+                    self.context.get_load_balancer_outbound_ports(),
+                    self.context.get_load_balancer_idle_timeout(),
+                    network_profile.load_balancer_profile,
+                )
+            else:
+                network_profile.load_balancer_profile = create_load_balancer_profile(
+                    self.context.get_load_balancer_managed_outbound_ip_count(),
+                    ipv6_count,
+                    self.context.get_load_balancer_outbound_ips(),
+                    self.context.get_load_balancer_outbound_ip_prefixes(),
+                    self.context.get_load_balancer_outbound_ports(),
+                    self.context.get_load_balancer_idle_timeout(),
+                )
+            print('post + ' + str(network_profile))
+
```

Comment:
We also have unit test cases for the `set_up_xxx` functions in `AKSPreviewDecorator`, please extend `test_set_up_network_profile` in file `test_decorator.py`.

Reply:
added unit tests in test_decorator and in test_aks_commands